Gallery-Server/src/main/java/edu/yu/cs/gallery/GalleryResource.java

```diff
@@ -41,8 +45,14 @@ public class GalleryResource {
     String hubURL;
 
     @GET
-    public Gallery getOnServer() {
-        return utility.gallery;
+        public Response getOnServer() {
+        return Response.status(Status.FOUND).entity(gr.findAll().firstResult()).build();
```

Comment:
curious - how come `FOUND` and on `OK`?

Reply:
Ok, what is the convention for the proper status here and in the method of getByID? Should it be OK or FOUND?

Reply:
I believe its OK. But you can just return the entity, and by default it will return 200 (OK)